<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>563</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>563</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Buscar archivo...</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logoibio.png</normaloff>logoibio.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{background-color:skyblue;}</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>11</y>
     <width>501</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Buscar el archivo con datos</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="btnBuscar">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{background-color:yellow;}

QPushButton:hover{background-color:orange;}</string>
      </property>
      <property name="text">
       <string>Buscar directorio</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="lineDirectorio">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QTreeWidget" name="treeArchivos">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <column>
       <property name="text">
        <string>Nombre del archivo</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nueva columna</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tipo de archivo</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tama√±o del archivo</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
